### ISSUE TITLE
Weatherman stopped working after August 19th

### ISSUE THREAD

**Issue by AseKarlsson** [chunkysteveo/OpenEPaperLink-HA-Weatherman#34]
Hi,
This code stopped working for me on 19th of August (cant remeber if I updated ePaper Integration or HA that day). 
When I run the HA automation I can see on the AP, that there in no new info comming from HA to be delivered to the display.
The automation itself does not return any error.
Other with same issue and a solution?

Thanks,
AseKarlsson

**Comment by chunkysteveo** [comment]
Is this still an issue?

**Comment by AseKarlsson** [comment]
Yes, still the same issue, I have not done anything with the automation since I reported problems.
Seems to get stucked at this point:

![Image](https://github.com/user-attachments/assets/5c3a2b9e-1ba5-4515-bf93-ba3e860eba0d)

**Comment by chunkysteveo** [comment]
Not had anyone else advise the automations have stopped. Are you able to start from scratch - the HA and epaper setup has changed a lot since then?

**Comment by AseKarlsson** [comment]
HA and OpenEPaper has of course been updated several times. I can start from scratch and test.

**Comment by chunkysteveo** [comment]
If you could, that would be great. There's not much info to go on to try to help troubleshoot this.

**Comment by AseKarlsson** [comment]
Now done this and the OpenEPaper tag got updated from HA, but it shows the weather from 19th of August and not from today and forward?

**Comment by AseKarlsson** [comment]
I have multiple weather forecast sensors and tried to add the below:

variables:
        forecast_entity: weather.home

But this only hives me errors, if this could be the issue?

**Comment by AseKarlsson** [comment]
Also tried to disable the second weather forecast Integration, but still showing the weather from the 19th of August

**Comment by AseKarlsson** [comment]
I hope I am into something here:

![Image](https://github.com/user-attachments/assets/48062ad9-e8c9-4420-b7d9-d11071ffb12e)

**Comment by chunkysteveo** [comment]
If it's 19th August - looks more like an issue with your templatensensor "Weatherman", like you've stopped it collecting the data on a time basis? Can you check what your Weatherman entitiy looks like in HA?

**Comment by AseKarlsson** [comment]
Change your code to this:

![Image](https://github.com/user-attachments/assets/eba9efbe-6b00-4deb-a5a8-3e1d8cd7acc4)

And it works again

**Comment by chunkysteveo** [comment]
Haha - glad you got there in the end!! Well done 👏🏻

**Comment by AseKarlsson** [comment]
![Image](https://github.com/user-attachments/assets/57a93353-e4d3-4dd6-ae16-b6c176f5e0cc)

Still seems to be off by 1 day!
18th is Friday not Thursday and so on

**Comment by chunkysteveo** [comment]
18 is the time... 18:00, not 18th. The day forecast is at the bottom of the display.

**Comment by AseKarlsson** [comment]
This is now clear, Thanks for the reply!

**Comment by chunkysteveo** [comment]
You can change the time and date format as you like too, using the date string formatting options. I use 6PM, 7PM etc. Good luck!

**Comment by AseKarlsson** [comment]
Thanks for feedback, but for me this is right as Europen, just mixed it with the date that was just a day off!
